tests/test_classifier/test_class_prediction_error.py

```diff
@@ -20,68 +21,97 @@
 
 import pytest
 import numpy as np
+import matplotlib.pyplot as plt
 
 from yellowbrick.classifier.class_balance import *
+from yellowbrick.exceptions import ModelError
 
-from tests.base import VisualTestCase
 from sklearn.svm import LinearSVC
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.datasets import make_multilabel_classification
+
+from tests.base import VisualTestCase
 
 ##########################################################################
 ## Data
 ##########################################################################
 
 X = np.array(
-        [[ 2.318, 2.727, 4.260, 7.212, 4.792],
-         [ 2.315, 2.726, 4.295, 7.140, 4.783,],
-         [ 2.315, 2.724, 4.260, 7.135, 4.779,],
-         [ 2.110, 3.609, 4.330, 7.985, 5.595,],
-         [ 2.110, 3.626, 4.330, 8.203, 5.621,],
-         [ 2.110, 3.620, 4.470, 8.210, 5.612,]]
-    )
+    [[2.318, 2.727, 4.260, 7.212, 4.792],
+     [2.315, 2.726, 4.295, 7.140, 4.783],
+     [2.315, 2.724, 4.260, 7.135, 4.779],
+     [2.110, 3.609, 4.330, 7.985, 5.595],
+     [2.110, 3.626, 4.330, 8.203, 5.621],
+     [2.110, 3.620, 4.470, 8.210, 5.612]]
+)
 
 y = np.array([1, 1, 0, 1, 0, 0])
 
 ##########################################################################
 ##  Tests
 ##########################################################################
 
+
 class ClassPredictionErrorTests(VisualTestCase):
 
-    @pytest.mark.skip(reason="not implemented yet")
-    def test_class_report(self):
+    def test_integration_class_prediction_error_(self):
         """
         Assert no errors occur during class prediction error integration
         """
         model = LinearSVC()
-        model.fit(X,y)
+        model.fit(X, y)
         visualizer = ClassPredictionError(model, classes=["A", "B"])
-        visualizer.score(X,y)
+        visualizer.score(X, y)
         self.assert_images_similar(visualizer)
 
-    @pytest.mark.skip(reason="not implemented yet")
-    def test_no_classes_provided(self):
+    def test_class_prediction_error_quickmethod(self):
```

Comment:
Just a quick glance at the quick method baseline image only shows one bar, was this the desired output?

Reply:
no it wasn't the desired outcome... I will take a look at it